sormas-base/setup/sormas.properties

```diff
@@ -62,6 +62,15 @@ app.url=
 # default: "/opt/sormas/sormas2sormas/"
 #sormas2sormas.path=/opt/sormas/sormas2sormas/
 
+# Key data for the generated SORMAS to SORMAS certificate";
+#sormas2sormas.keyAlias=
+#sormas2sormas.keystoreName=
+#sormas2sormas.keystorePass=
+
+# SORMAS to SORMAS truststore data";
+#sormas2sormas.truststoreName=${TRUSTSTORE_FILE_NAME}";
```

Comment:
I guess the value should be removed? The training `"}` is also present in the two headers above.